### ISSUE TITLE
Todays pdate broke csg-hooker

### ISSUE THREAD

**Issue by kusayuzayushko** [Teklad/csgo-horker#81]
Can you do something with it?

**Comment by Teklad** [comment]
I'll look into it over the weekend.  There's no telling what all has changed with a massive update like that.

**Comment by Teklad** [comment]
Mostly fixed now.  Auto-trigger is still broken for now, but I'll fix that this weekend.

**Comment by Teklad** [comment]
The glow is fixable... I've just not poked at battle royale mode yet to find out how to fix it.

**Comment by koutsie** [comment]
They changed movement so much.... 

Relax, take it easy.

**Comment by RealBloodman** [comment]
Forgot to mention aimbot doesn't work either on the new gamemode

**Comment by RealBloodman** [comment]
Yup tried ally aimbot+glow and it worked fine, only problem is bhop so..yeah

**Comment by Teklad** [comment]
What do you mean by sources?
